packages/app/app/components/Playlists/PlaylistsHeader/index.tsx

```diff
@@ -5,28 +5,52 @@ import { Button } from '@nuclear/ui';
 import Header from '../../Header';
 import styles from './styles.scss';
 import { Icon } from 'semantic-ui-react';
+import InputDialog from '../../InputDialog';
 
 type PlaylistsHeaderProps = {
   showText: boolean;
   handleImportFromFile: React.MouseEventHandler;
+  createNew: (name: any) => void;
 }
 
-const PlaylistsHeader: React.FC<PlaylistsHeaderProps> = ({ showText, handleImportFromFile }) => {
+const PlaylistsHeader: React.FC<PlaylistsHeaderProps> = ({ showText, handleImportFromFile, createNew }) => {
   const { t } = useTranslation('playlists');
 
+  const handleAddPlaylist = (name) => {
+    createNew(name);
+  };
+
   return (
     <div className={styles.header_container}>
       {showText && <Header>{t('header')}</Header>}
       {!showText && <span />}
 
-      <Button
-        basic
-        onClick={handleImportFromFile}
-        data-testid='import-from-file'
-      >
-        <Icon name='file text' />
-        {t('import-button')}
-      </Button>
+      <div>
```

Comment:
You can get rid of this div

Reply:
no, I can't because header has flex with space between